scaffolder-skeletons/catalog-info-skeleton/catalog-info.yaml

```diff
@@ -18,7 +18,11 @@ metadata:
     {%- endif %}
     backstage.io/kubernetes-id: ${{ values.repoName }}
     backstage.io/techdocs-ref: dir:.
-    github.com/project-slug: ${{ values.githubOrg }}/${{ values.repoName }}
+    {%- if values.sourceControl == "gitlab.com" %}
```

Comment:
Would it make sense to do it more generic? Something like 
 
```yaml
{%- if values.sourceControl %}
    ${{values.sourceControl}}/project-slug: ${{ values.orgName }}/${{ values.repoName }}
{%- endif %}
```

Reply:
Ideally that would be nice; however, `${{values.sourceControl}}/project-slug:` is not standardized for the source control plugins. e.g. [Azure DevOps](https://github.com/backstage/backstage/blob/master/plugins/azure-devops/README.md#entity-annotation)

Reply:
uff, ok. This if-else block is going to look "interesting" as we add more and more :-/ , guess not much we can do about it than.